.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] GOOD JOB SO FAR 🎉😉. I think you forgot to add node modules inside `.gitignore`. Please kindly add also `node_modules` in the 2nd line to fix the issue.